WordPress/Classes/ViewRelated/Post/PostEditorState.swift

```diff
@@ -51,6 +51,21 @@ public enum PostEditorAction {
         }
     }
 
+    fileprivate var publishingActionQuestionLabel: String {
```

Comment:
So we only show the publish confirmation for 'async' publish actions (that is, publish actions that close the editor – see [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/38c13a0be0115f943677a2eba2c3f23a3e108de0/WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift#L1201)). 

It's probably best if we don't request localisations for strings we're not going to use, so we should probably only return values for `.publish, .publishNow, .schedule, .submitForReview`. You could remove the extra cases, return an empty string in the `default` case (or we could make the return value optional), and perhaps add a comment explaining that the dialog is only shown for async publishing methods.

Reply:
@frosty I think that the showing the for values with a default blank string sounds like a good idea. Nothing against optionals. I've separated out `.publish` and `.publishNow` in my switch cases just so they're not displaying the same message and button.